src/schema.py

```diff
@@ -158,9 +158,13 @@ def resolve_account_info(self, info, session_id=None):
       else:
         name = location
 
+      # Transaction Type 3 denotes payments that increase balance
+      positive = txn['transactionType'] == 3
```

Comment:
let's make a constant for this. like `POSITIVE_TRANSACTION_TYPE = 3` or something similar and add it to the constants.py!